adb_graphics/datahandler/grib.py

```diff
@@ -155,6 +167,9 @@ def get_level(self, field, level, spec):
         levs = self.ds[dim_name].values
 
         # Requested level
+        if level == 'mean':
+            lev = 0
+            return lev
```

Comment:
These lines should not be needed. Provide `vertical_index: 0` in the default_specs.yml instead.

Reply:
I did try that and it does work, but I decided to handle it in grib.py rather than having a vertical_index parameter in the default_specs simply because it looks confusing when just looking at the default_specs.

Reply:
I think a quick comment in the default_specs.yml will help with confusion, and keep a bunch of level-type logic out of the code and in the config files.